Library/Homebrew/hardware.rb

```diff
@@ -55,6 +55,8 @@ def self.cores_as_words
     when 1 then "single"
     when 2 then "dual"
     when 4 then "quad"
+    when 6 then "hexa"
```

Comment:
From Amazon it looks like the most common usage is "hex core" and "octa core"; "hexa core" returned no results for me and tried to autocorrect.